GSA/uswds-sf-lightning-community
# Issue 139

Addresses #139 


Explain the **motivation** for making this change. What existing problem does the pull request solve?



**Test plan (required)**

Demonstrate the code is solid.

- automated tests were not available due to the presence of `navigationMenuBase`.
- Manual test steps
  - Deploy code
  - Update Default Navigation for a test Community
  - Make one or more labels for the Navigation Menu to be lengthy (40+ characters)
  - [ ] Toggle off Wrap (No). Change viewport to <500px. Observe that the lengthy breadcrumb displays on a new line
  - [ ] Toggle on Wrap (Yes). In a narrow viewport, observe that text is truncated with `...`
  - [ ] Update Page Title, observe value appears at the end of the breadcrumb and is not a link
  - [ ] Create new Navigation Menu, update Navigation Menu text field with name of new menu. Observe new breadcrumb values

**Code formatting**

👍 

**Closing issues**

closes #139
